modules/network-observability-working-with-conversations.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-working-with-conversations_{context}"]
+= Working with conversation tracking
+As an administrator, you can you can group network flows that are part of the same conversation. A conversation is defined as a grouping of peers that are identified by their IP addresses, ports, and protocols, resulting in an unique *Conversation Id*. You can query conversation events in the web console. These events are represented in the web console as follows:
+
+** *Conversation start* happens when a connection is starting or TCP flag intercepted
+** *Conversation tick* occurs at every specified interval defined in flow collector `spec.processor.conversationHeartbeatInterval` while the connection is up.
+** *Conversation end* occurs when the `FlowCollector` `spec.processor.conversationEndTimeout` is reached or TCP flag intercepted.
+** *Flow* is the network traffic flow that occurs within the specified interval.
+
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *Flow Collector*. 
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` so that `spec.processor.logTypes`, `conversationEndTimeout`, and `conversationHeartbeatInterval` are set according to your observation needs. A sample configuration is as follows:
++
+[id="network-observability-flowcollector-configuring-conversations_{context}"]
+.Configure `FlowCollector` for conversation tracking
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+ processor:
+  conversationEndTimeout: 10s                  <1>
+  logTypes: FLOWS                              <2>
+  conversationHeartbeatInterval: 30s           <3>
+----
+<1> The *Conversation end* event represents the point when the `conversationEndTimeout` is reached or the TCP flag is intercepted.
+<2> When `logTypes` is set to `FLOWS`, only the *Flow* event is exported. If you set the value to `ALL`, the *Conversation start*, *Heartbeat*, *Conversation end*, and *Flow* events are exported and visible in the *Network Traffic* page. To focus only on conversation events, you can specify `CONVERSATIONS` which exports the *Conversation start*, *Heartbeat* and *Conversation end* events; or `ENDED_CONVERSATIONS` exports only the *Conversation end* events. Storage requirements are highest for `ALL` and lowest for `ENDED_CONVERSATIONS`.
```

Comment:
Rather than mentioning If you set the value to `ALL`, the *Conversation start*, *Heartbeat*, *Conversation end*, and *Flow* events are exported and visible in the *Network Traffic* page.
how about 
If you set the value to `ALL`, both conversation and Flow events are visible and can be switched in the Query options dropdown or side panel.
The previous event sounds like all 4 event types coexist which is not the case, if you get what I am saying

Reply:
I see what you mean. I'll take it. Its cuts down on the wordiness here as well :)